src/routes/Birthday.ts

```diff
@@ -0,0 +1,168 @@
+import moment from 'moment';
+import { CronJob } from 'cron';
+import { Permissions, TextChannel } from 'discord.js';
+import { User } from '../entity/User';
+import Router from '../Router';
+
+const router = new Router();
+
+router.use('set', async ({ msg, user }) => {
+  const prefix = 'ei bday set';
+  const args = msg.content.slice(prefix.length).trim().split('/');
```

Comment:
```suggestion
router.use('set', async ({ msg, user, params }) => {
  const rawDate = param[0]
  
  if (typeof(rawDate) !== 'string') {
    msg.channel.send('Ik verwacht een datum als argument');
    
    return;
  }
  
  const args = msg.content.slice(rawDate).trim().split('/');
```
Hier kan je de params array gebruiken